src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -195,16 +197,18 @@ Clock getClock() {
     }
 
     @Override
-    protected boolean isMtTable(String tableName) {
-        return mtTables.containsKey(tableName) && !tableName.startsWith(backupTablePrefix);
+    protected boolean isPhysicalTable(String tableName) {
```

Comment:
Again, this is now based on table prefix, instead of looking at the set of static physical tables, because now we have physical tables created on the fly. Need to exclude the table description repo table because that also has the same prefix but isn't a data table.

Reply:
Is this filter going to match the `*.Leases` tables?

Reply:
Oh yeah it is going to match that too... Maybe it'd be better to make dynamically created tables have an extra suffix to the prefix, e.g., [clientTablePrefix].d, or something like that, and look for only the static data tables and dynamic data tables found in this way.

What are the rules for mt-dynamo table names and prefixes? Are they defined somewhere?